sdk/src/types/block/semantic/mod.rs

```diff
@@ -221,6 +221,14 @@ impl<'a> SemanticValidationContext<'a> {
             }
         }
 
+        // Add allotted mana
+        for mana_allotment in self.transaction.allotments() {
```

Comment:
Super nit picking here but can you put it right after the outputs loop instead of before pls?

Reply:
Why? In the tx the allotments are before outputs (I'm still fine with changing it, also initially thought about moving it below, only did not because of the tx structure)

Reply:
No worries, just works better in my head for some reason, ignore me.